src/app/public/modules/action-hub/action-hub.component.ts

```diff
@@ -17,30 +18,78 @@ let data: SkyActionHubData;
 export class SkyActionHubComponent {
   /**
    * Passes a SkyActionHubData object to build the action hub. The page loads until
-   * `title` has a value.
+   * `[data]` has a value.
    *
    * @param value
    */
   @Input()
   public set data(value: typeof data) {
-    this.needsAttention = value.needsAttention;
-    this.parentLink = value.parentLink;
-    this.recentLinks = value.recentLinks;
-    this.relatedLinks = value.relatedLinks;
-    this.title = value.title;
+    if (typeof value === 'undefined') {
```

Comment:
Is there a reason that we are looking specifically for `undefined` here and not just for any falsey value? I'm thinking specifically about if the rare scenario of `null` came in here.